quickwit/quickwit-cli/src/main.rs

```diff
@@ -130,7 +135,11 @@ async fn main() -> anyhow::Result<()> {
     #[cfg(feature = "jemalloc")]
     start_jemalloc_metrics_loop();
 
-    setup_logging_and_tracing(command.default_log_level(), &build_info)?;
+    setup_logging_and_tracing(
+        command.default_log_level(),
+        !matches.is_present("no-ansi"),
```

Comment:
```suggestion
        !matches.is_present("no-clor"),
```

Reply:
Have changed this, thought it would make sense to keep it as ansi but it ended up looking more confusing 😅